petergreve/application/app/key.js

```diff
@@ -1,3 +1,3 @@
-const key = 'AIzaSyB6LYUvbFC73pLDjn-FhpcEvSgJ48f3Ag4'
+const key = ''
```

Comment:
You should have reset the commit first and then force push. 

You can still do it, like so:
```
git reset HEAD~2 --soft
git commit -m "<some relevant msg here>"
git push --force
```
And I have to say it again, try to use `--force` as seldom as possible. As a rule of thumb: `Don't use the force!`